datalad/ui/base.py

```diff
@@ -31,10 +31,21 @@ def question(self, text,
         pass
 
     def yesno(self, *args, **kwargs):
+        # Provide some default sugaring
+        default = kwargs.pop('default', None)
+        if default is not None:
+            if default in {True}:
```

Comment:
If you're going to have only one value, why not `is True` here and `is False` below?

Reply:
I find it less readable because I'm not used to seeing it.  As for the logic, I don't consider changing `is True` to `in {True, X}` if the need arises to be a maintenance burden.  And it seems you could use the same logic for pretty much any `is ATOM`, or any `== ATOM` for that matter.  Anyway, my comment was a "just curious" one as well.